static/js/components/FollowupRequestLists.jsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
```

Comment:
Should rename this file Lists -> List

Reply:
i have it as lists as it renders a different list for each instrument, since each instrument has a different schema and therefore a frontend representation with different table columns.